src/services/import-manifest-service.ts

```diff
@@ -0,0 +1,81 @@
+import { ListrTaskWrapper } from 'listr2';
+
+import { importAppManifestUrl } from 'consts/urls';
+import { execute } from 'services/api-service';
+import { compressFilesToZip, readZipFileAsBuffer } from 'services/files-service';
+import { PromptService } from 'services/prompt-service';
+import { ImportCommandTasksContext } from 'types/commands/manifest-import';
+import { AppId, AppVersionId } from 'types/general';
+import { HttpMethodTypes } from 'types/services/api-service';
+import { appsUrlBuilder } from 'utils/urls-builder';
+
+export const uploadManifestTsk = async (
+  ctx: ImportCommandTasksContext,
+  task: ListrTaskWrapper<ImportCommandTasksContext, any>,
+) => {
+  task.output = `Zipping your manifest file`;
+  const zipFilePath = await compressFilesToZip([ctx.manifestFilePath]);
+  const buffer = readZipFileAsBuffer(zipFilePath);
+  task.output = `Uploading your app manifest`;
+  await uploadZippedManifest(buffer, { appId: ctx.appId, appVersionId: ctx.appVersionId });
+  task.output = `your app manifest has been uploaded successfully`;
```

Comment:
Any reason we're using a zipped version?

Reply:
The manifest endpoints support only zip format (in oder to support in feature dependencies and assets such as images etc...)